src/DetailsView/components/settings-panel/settings/settings-provider-impl.ts

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { BugFilingSettings } from './bug-filing/bug-filing-settings';
+import { HighContrastSettings } from './high-contrast/high-contrast-settings';
+import { create } from './settings-provider';
+import { TelemetrySettings } from './telemetry/telemetry-settings';
+
+export const SettingsProviderImpl = create([TelemetrySettings, HighContrastSettings, BugFilingSettings]);
```

Comment:
Do you think we can rename `create` to maybe tell what we are creating? Personally, `create` looks a little too general for me. Let me know what you think